dot-net-sdk/EppoClient.cs

```diff
@@ -400,6 +402,45 @@ public static EppoClient Init(EppoClientConfig eppoClientConfig)
         return new(variation);
 
     }
+
+    public void LogNonBanditAction(string subjectKey,
```

Comment:
Could we allow the user to pass the variation assigned for this context?
We compute it below, but that ultimately requires 1) a default value of some kind (see above) and 2) would duplicate the log entry for this assignment.

Reply:
This functionality is still a bit wishy-washy. It seems like it's not even implemented in the Python SDK.

The Java SDK computes the variation to log as the bandit key. But that may not always be correct if the user is not assigned a bandit.

I think either we:
1. Just get the bandit at play from the mapping of flags to bandits (if many, perhaps just take the the first one?)
2. Drop this functionality entirely

I'll start a discussion in the `#bandits` channel